specification/definitions/Address.json

```diff
@@ -1,20 +1,17 @@
 {
   "type": "object",
   "required": [
-    "street",
-    "number",
-    "postal_code",
+    "addressline_1",
     "city",
-    "cc"
+    "country_code"
   ],
   "properties": {
-    "street": {
+    "addressline_1": {
       "type": "string",
-      "example": "Hoofdweg"
+      "example": "Hoofdweg 679"
     },
-    "number": {
-      "type": "string",
-      "example": "679"
+    "addressline_2": {
```

Comment:
Yes, we moeten sowieso nog meer van de  attributes bespreken. Maar wel handig om deze alvast te mergen, zodat we er makkelijker over kunnen praten met swagger erbij.